src/cma.ts

```diff
@@ -225,26 +156,45 @@ async function getCmaOutput(readLine, errorObj) {
   });
 }
 
+/**
+ * Invoke the task function and wrap the result as a Promise
+ *
+ * @param {Function} TaskFunction - the task function to be invoked
+ * @param {Object} cumulusMessage - a full Cumulus message
+ * @param {Object} context - the Lambda context
+ * @returns {Promise} - the result of invoking the task function
+ */
+function invokePromisedTaskFunction(
+  TaskFunction: (msg: LoadNestedEventInput, context: Context) => Promise<unknown>,
```

Comment:
Consider pulling the function type out into its own named type.

Reply:
b4ac934